app/Console/Commands/Takedate.php

```diff
@@ -45,22 +50,36 @@ public function handle(Extractor $metadataExtractor)
 		$i = $from;
 		foreach ($photos as $photo) {
 			$url = Storage::path('big/' . $photo->url);
-			if (file_exists($url)) {
-				$info = $metadataExtractor->extract($url, $photo->type);
-				if ($info['takestamp'] != null) {
-					$photo->takestamp = $info['takestamp'];
-					if ($photo->save()) {
-						$this->line($i . ': Takestamp updated for ' . $photo->title);
-					} else {
-						$this->line($i . ': Failed to update takestamp for ' . $photo->title);
-					}
+			$i++;
```

Comment:
This `$i++` is not consistent with what is actually being done and is off by +1.
Just place it before the `continue` and at the end of the loop.